src/state.rs

```diff
@@ -40,6 +40,7 @@ pub mod workload;
 pub struct Upstream {
     pub workload: Workload,
     pub port: u16,
+    pub sans: Vec<String>,
```

Comment:
If this is always empty vector (didn't see any place where san is being added to it) why do you need it?

Reply:
thanks - this snuck in during that latest merge with latest